common/config/rush/pnpm-lock.yaml

```diff
@@ -1,43 +1,44 @@
 lockfileVersion: 5.4
 
 importers:
+
```

Comment:
why do we have 7927 lines changes for this file ?

Reply:
i think this is because of format.